pkgs/by-name/me/met/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+let
+  version = "0.0.5";
+in
+buildGoModule {
+  pname = "met";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "jaxxstorm";
+    repo = "met";
+    tag = "v${version}";
+    hash = "sha256-nfI0sJSEsKTQZkM5grUBdC7PkzTapEgF1zYzIdkoN2I=";
+  };
+
+  vendorHash = "sha256-dl90ItX62FfDAzAHHBxp4ojegGCj2oMKl47BjsD2RU0=";
+
+  ldflags = [
+    "-X=main.Version=${version}"
+  ];
+
```

Comment:
add versionCheckHook and nix-update-script

Reply:
I've added the versionCheckHook but I don't tend to add nix-update-script unless it needs specific configuration e.g. skipping tags.